src/services/accounts/AccountsAssetsService.spec.ts

```diff
@@ -81,9 +81,9 @@ const assetBalanceObjThree = {
 };
 
 const assetBalanceFactory = {
-	'10': assetBalanceObjOne as PalletAssetsAssetBalance,
-	'20': assetBalanceObjTwo as PalletAssetsAssetBalance,
-	'30': assetBalanceObjThree as PalletAssetsAssetBalance,
+	'10': assetBalanceObjOne as unknown as PalletAssetsAssetBalance,
```

Comment:
can you explain why the `unknown` type casts are needed?

Reply:
Yea because the typescript compiler doesnt have the ability to cast `Obj A` as `Obj B` since their contents are not the exact same type structure. `PalletAssetsAssetBalance` has a `extra` field of type Null that we don't add for this specific test because it's not needed.